plugin/db/driver.go

```diff
@@ -215,28 +215,28 @@ type MigrationInfoPayload struct {
 
 // MigrationInfo is the API message for migration info.
 type MigrationInfo struct {
```

Comment:
nit: didn't comment on the previous PR, but I suggest not to include the migrationInfo to payload but a curated subset of it.

Reply:
SG.

Reply:
Second thought: I would like to leave it for now, because this would introduce much more changes.
VCS push event will call `ParseMigrationInfo` to parse a subset of the MigrationInfo, we need to split this out, which seems not quite necessary for now.

Reply:
I think @d-bytebase meant create a new struct to store just necessary information to the `payload`, not splitting `MigrationInfo` itself. Something like

Reply:
Yup, @unknwon is correct. Honest, I think we should do that before this change before we get corrupted data.